src/main/java/mekanism/common/integration/computer/BoundMethodHolder.java

```diff
@@ -23,6 +37,11 @@ public abstract class BoundMethodHolder {
 
     protected Lazy<String[]> methodNames = Lazy.of(()->this.methods.keys().toArray(new String[0]));
 
+    public BoundMethodHolder() {
```

Comment:
This can just be `protected` as it is an abstract class so everything has to implement it anyways if it wants to be called.